### ISSUE TITLE
Open Link to different website on the same domain?

### ISSUE THREAD

**Issue by jgivensNAFCPL** [zebradog/kiosk#21]
Hello again, I think this would be considered an "enhancement" request, but would it be possible to be able to modify the settings so that, as long as the links that are clicked all point to pages that are on the domain that they will open within the Kiosk app? I'm looking into Chromeboxes as replacements for our library's existing public access catalog PCs and we have two catalog URLs that need to be accessed from the PAC itself.

Thank you for your time!

**Comment by matt-cook** [comment]
Are these 'pop-up' type links? It should be possible to use links from the website you specify to any domain inside the primary frame. Currently pop-ups and new windows are being blocked: #6 .

**Comment by jgivensNAFCPL** [comment]
No sir, these are links to a different page on the domain, for example, if our teen/adult PAC URL is http://URL.org then our children's PAC URL is http://URL.org/kids . If you click on a link to either one of them (depending on the PAC URL you're currently looking at, obviously) I'd love the Kiosk app to be able to go to the other page.

Hopefully that clarifies my request, thank you!

**Comment by matt-cook** [comment]
Apologies if I'm being dense.

You need a system to redirect requests? for example: 'if request.url == '
http://URL.org' then goto 'http://URL.org/kids' for all links on that
domain?

**Comment by jgivensNAFCPL** [comment]
Sorry, it's not you I'm just not explaining my request very well. So on each of these pages, in the lower left hand corner there is a link to webpage for the other PAC URL. For example, if you are on the teen/adult PAC webpage and scroll to the bottom, in the left corner there will be a link that states "Children's Catalog" (and it is a link to http://URL.org/kids) . If you click that link, the window refreshes (the window that is currently open, it does not open a new window) and the Children's PAC URL is displayed. On the Children's PAC webpage, if you scroll down and go to the same area of the screen it will say "Library Catalog" (and it is a link to http://URL.org). If you click that link, it takes you back to the teen/adult PAC webpage. And so on.

Basically, I'm wanting to go back and forth between those two pages. I know it breaks the basic principles of what a kiosk is supposed to do, but it's important to the department managers that both PACs are accessible no matter what part of the library you are in. I figured it couldn't hurt to ask if that was possible.

Thank you for your time!

**Comment by matt-cook** [comment]
This should probably done as a wrapper URL for both pages: it could load the correct page in an iframe with the link to the alternate (and then swap), or simply add the links directly to a modified version of each application.

If you do need the feature built-in contact support@zebradog.com for a custom version. 

tldr; best done in the content versus a function of the Kiosk app.

**Comment by jgivensNAFCPL** [comment]
Hi Matt, I ended up taking a different approach: http://community.spiceworks.com/people/randalljgivens/projects/public-access-catalog-refresh

It was a fun project, overall, and, while the Chromebox itself is an amazing little device, I'm not a fan of Chrome OS itself, so if these little guys every need to be repurposed I can simply install a different flavor or Linux.

I appreciate you looking into this though, but I think we are good now. Thanks again!
